docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,136 @@
+# Getting Started
+
+## What you'll do
+
+- Deploy the [tmate-ssh-server charm](https://charmhub.io/tmate-ssh-server)
+- Get the .tmate.conf through `get-server-config` action
+- Create a tmate client machine and configure tmate client
+- SSH into tmate terminal
+
+Tmate is a remote terminal sharing tool that allows users to securely share their terminal with
+others in real-time, making it easy to collaborate, troubleshoot, and provide support across
+different systems and networks. It provides a seamless way to establish SSH connections and enables
+remote access without requiring complex network configurations.
+
+### Prerequisites
+
+To deploy a tmate-ssh-server charm, you will need a juju bootstrapped with any machine controller.
```

Comment:
You will need a juju? Perhaps the wording could be refined?

Reply:
Updated to: `a Juju controller bootstrapped with any machine type controller`. Thank you :D